ParameterStore.fs

```diff
@@ -57,7 +58,7 @@ open Amazon.SimpleSystemsManagement
             getParametersStoreParameters()
           with
             | ex ->
-                tracefn "Failed to get parameter store parameters %s" ex.Message
+                failwithf "Failed to get parameter store parameters %s" ex.Message
```

Comment:
we don't really want to fail here, it's intentional that we just trace. this is due to the fact that we can run this on a machine w/o amazon credentials and the build would still pass